cmd/kpod/create.go

```diff
@@ -183,6 +192,52 @@ func createCmd(c *cli.Context) error {
 	return nil
 }
 
+const seccompDefaultPath = "/etc/crio/seccomp.json"
+
+func parseSecurityOpt(config *createConfig, securityOpts []string) error {
+	var (
+		labelOpts []string
+		err       error
+	)
+
+	for _, opt := range securityOpts {
+		if opt == "no-new-privileges" {
+			config.noNewPrivileges = true
+		} else {
+			var con []string
+			if strings.Contains(opt, "=") {
```

Comment:
Shouldn't need this check. If Strings.SplitN doesn't find the splitting character, it will just return an array of length 1.
And you are already checking for the length in the next line.

Reply:
Fixed